packages/next-swc/crates/next-core/js/src/entry/app/route-bootstrap.ts

```diff
@@ -1,40 +1,50 @@
+// PAGE is set from rust code
 declare const PAGE: string;
+// PATHNAME is set from rust code
+declare const PATHNAME: string;
 
+import { HandlerProvider } from "next/dist/build/webpack/loaders/next-edge-app-route-loader/provider";
 import { adapter, enhanceGlobals } from "next/dist/server/web/adapter";
-import { getHandle } from "next/dist/build/webpack/loaders/next-edge-app-route-loader/handle";
 
+// For each of the route kinds, import the route so we can pass it to the
+// HandlerProvider.
+// @ts-expect-error - ROUTE is set from rust code
+import { Route } from "ROUTE";
+
+import { requestAsyncStorage } from "next/dist/client/components/request-async-storage";
 import { staticGenerationAsyncStorage } from "next/dist/client/components/static-generation-async-storage";
+
+import * as headerHooks from "next/dist/client/components/headers";
 import * as serverHooks from "next/dist/client/components/hooks-server-context";
 import { staticGenerationBailout } from "next/dist/client/components/static-generation-bailout";
```

Comment:
Do we need all these imports here, or could they be moved into `next/dist/build/webpack/loaders/next-app-loader/routes/app-route.ts`